integrations/eureka/eureka/src/main/java/io/helidon/integrations/eureka/EurekaRegistrationFeature.java

```diff
@@ -0,0 +1,1039 @@
+ *
+ *       <td>If the {@linkplain WebServer#hasTls() webserver has TLS enabled}, the default value is {@code 80}. If the
+ *       webserver does not have TLS enabled, the default value is the port on which the webserver is currently running</td>
+ *
+ *       <td>Eureka makes a distinction between a <dfn>port</dfn> and a <dfn>secure port</dfn>. Both values
+ *       are registered, even if one of the two is not applicable.</td>
+ *
+ *     </tr>
+ *
+ *     <tr>
+ *
+ *       <th scope="row">{@code eureka.instance.securePort}</th>
+ *
+ *       <td>{@code int}</td>
+ *
+ *       <td>The <dfn>secure port</dfn> to be registered for this service instance.</td>
+ *
+ *       <td>If the {@linkplain WebServer#hasTls() webserver has TLS enabled}, the default value is the port on which the webserver is currently running. If the
+ *       webserver does not have TLS enabled, the default value is {@code 443}.</td>
+ *
+ *       <td>Eureka makes a distinction between a <dfn>port</dfn> and a <dfn>secure port</dfn>. Both values
+ *       are registered, even if one of the two is not applicable.</td>
+ *
+ *     </tr>
+ *
+ *     <tr>
+ *
+ *       <th scope="row">{@code eureka.instance.traffic.enabled}</th>
+ *
+ *       <td>{@code boolean}</td>
+ *
+ *       <td>Whether the service instance is able to respond to requests upon registration.</td>
+ *
+ *       <td>{@code true}</td>
+ *
+ *       <td>If this value is set to {@code false}, then a call must be made out of band, normally by a health check
+ *       mechanism, to the {@link #markUp()} method to report to Eureka that the service instance is able to respond to
+ *       requests.</td>
+ *
+ *     </tr>
+ *
+ *     <tr>
+ *
+ *       <th scope="row">{@code eureka.instance.lease.renewalInterval}</th>
+ *
+ *       <td>{@code int}</td>
+ *
+ *       <td>The duration, in seconds, between registration (lease) renewal attempts.</td>
+ *
+ *       <td>{@code 30}</td>
+ *
+ *       <td></td>
+ *
+ *     </tr>
+ *
+ *     <tr>
+ *
+ *       <th scope="row">{@code eureka.instance.lease.duration}</th>
+ *
+ *       <td>{@code int}</td>
+ *
+ *       <td>The duration, in seconds, of a successful registration (lease).</td>
+ *
+ *       <td>{@code 90}</td>
+ *
+ *       <td></td>
+ *
+ *     </tr>
+ *
+ *     <tr>
+ *
+ *       <th scope="row">{@code eureka.instance.metadata}</th>
+ *
+ *       <td>{@link Map Map&lt;String, String&gt;}</td>
+ *
+ *       <td>{@link String}-typed key-value pairs describing metadata to accompany a service instance registration.</td>
+ *
+ *       <td>none</td>
+ *
+ *       <td></td>
+ *
+ *     </tr>
+ *
+ *   </tbody>
+ *
+ * </table>
+ *
+ * <h2>Logging</h2>
+ *
+ * <p>The {@link Logger} used by instances of this class is named {@code io.helidon.integtrations.eureka.EurekaRegistrationFeature}.</p>
+ *
+ * @see #afterStart(WebServer)
+ */
+public final class EurekaRegistrationFeature implements HttpFeature {
```

Comment:
Please reformat the class with the rules we have for Idea. If you do not use Idea, please put public methods first, package local next, private last.

Reply:
OK; so not alphabetical then. Will fix.